game/scripts/npc/abilities/sandking_sand_storm.txt

```diff
@@ -21,11 +21,11 @@
     // Casting
     //-------------------------------------------------------------------------------------------------------------
     "AbilityCastRange"                                    "0"
-    "AbilityDuration"                                     "20 25 30 35 45 50"
+    "AbilityDuration"                                     "20 25 30 35 40 45"
 
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "40 34 28 22 18 16"
+    "AbilityCooldown"                                     "40" //OAA
```

Comment:
This spell is so cancer and not only in OAA. Duration and cooldown are scaled terribly. At the time I had no idea how to make Sand Storm uptime more balanced and less annoying. I though this would do until vanilla dota balance patch comes.

Reply:
40 seconds is so extreme. This hero isn't azmazing if you actually itemise around it, also you can abuse the fact that he is forced into an extremly linear item build.